src/Services/BoilerplateSettingsFactory.cs

```diff
@@ -30,6 +35,23 @@ public BoilerplateSettingsFactory()
 		{
 			this.dte = (DTE2)ServiceProvider.GlobalProvider.GetService(typeof(DTE));
 			this.personalSettings = new BoilerplateSettings(this.personalStore);
+
+			SettingsManager settingsManager = new ShellSettingsManager(ServiceProvider.GlobalProvider);
+			WritableSettingsStore store = settingsManager.GetWritableSettingsStore(SettingsScope.UserSettings);
+			store.CreateCollection(PersonalBoilerplateSettingsStore.CollectionPath);
+			if (store.PropertyExists(PersonalBoilerplateSettingsStore.CollectionPath, UserBoilerPlateSettings))
+			{
+				UserCreatedSettingsPath = store.GetString(PersonalBoilerplateSettingsStore.CollectionPath, UserBoilerPlateSettings);
+
+				if (string.IsNullOrEmpty(UserCreatedSettingsPath) == false)
+				{
+					LoadUserCreatedSettings = true;
+				}
+				else
+				{
+					LoadUserCreatedSettings = false;
+				}
```

Comment:
this.LoadUserCreatedSettings = !string.IsNullOrEmpty(this.UserCreatedSettingsPath)

OR make LoadUserCreatedSettings a get-only property that returns !string.IsNullOrEmpty(this.UserCreatedSettingsPath) so that it doesn't require updating when UserCreatedSettingsPath is updated.